external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/Timer.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.storm.kafka.spout.internal;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by liurenjie on 22/12/2016.
```

Comment:
Sorry for that. I've fixed it. I'll merge your change later after you submitted it.